Example/AblyChatExample/ContentView.swift

```diff
@@ -134,12 +134,14 @@ struct ContentView: View {
                     }
             }
         }
-        .tryTask { try await setDefaultTitle() }
-        .tryTask { try await attachRoom() }
-        .tryTask { try await showMessages() }
-        .tryTask { try await showReactions() }
-        .tryTask { try await showPresence() }
-        .tryTask { try await showOccupancy() }
+        .tryTask {
+            try await setDefaultTitle()
```

Comment:
Now it's `Task { Task {...} }`

Reply:
What do you see as being the issue with that?

Reply:
Not an issue, but a bit of extra thread jumps, but probably it's optimized by compiler/runtime, so I'm resolving..